### ISSUE TITLE
Media tracks

### ISSUE THREAD

**Issue by lukehedger** [exogen/graphbrainz#77]
Is it possible to retrieve all of the `tracks` on a `Release`? Looking at the MusicBrainz API it should be possible to do this:
```gql
{
  lookup {
    release(mbid: "617908f2-7a0b-3aad-ae02-5a5c125fa1a7") {
      media {
        trackCount
        position
        tracks # Cannot query field "tracks" on type "Medium"
        formatID
        format
      }
    }
  }
}
```

This is the equivalent request on the JSON web service: http://musicbrainz.org/ws/2/release/617908f2-7a0b-3aad-ae02-5a5c125fa1a7?inc=recordings&fmt=json
```json
{
  "id": "617908f2-7a0b-3aad-ae02-5a5c125fa1a7",
  "media": [
    {
      "track-count": 12,
      "position": 1,
      "track-offset": 0,
      "tracks": [{}],
      "format-id": "9712d52a-4509-3d4b-a1a2-67c88c643e31",
      "format": "CD"
    }
  ]
}
```

**Comment by exogen** [comment]
Not presently, but that is a great TODO item. I don't know much about tracks – I assume it always returns all tracks, and not a subset where you have to use `browse` somehow to get the full list? Trying to decide if it should be a Connection or not.

**Comment by lukehedger** [comment]
Yep - it just returns all tracks, as far as I can tell. As for pagination, reckon that would make sense and I guess that is the purpose of the `track-offset` field?

**Comment by exogen** [comment]
Released in v8.1.0!
